src/router/routes.js

```diff
@@ -7,6 +7,14 @@ export default [
 			contentfulPage: () => 'home',
 		},
 	},
+	{
+		path: '/homepage-classic',
+		name: 'homepage',
+		component: () => import('@/pages/Homepage/HomepageClassic'),
+		meta: {
+			contentfulPage: () => 'homepage',
```

Comment:
The value returned here is the page key in contentful. Maybe we should make this `home-classic`  for a more clear distinction.